lib/index.js

```diff
@@ -1,7 +1,7 @@
 "use strict"
+import core from 'spec-url'
+import { pct } from 'spec-url/src/pct.js'
 const log = console.log.bind (console)
-const core = require ('spec-url')
-const { pct } = require ('spec-url/src/pct')
```

Comment:
Also this `pct` seems to be currently unused. It's available as `core.pct` anyway. Should I remove it?